pkgs/development/libraries/tdlib/default.nix

```diff
@@ -0,0 +1,23 @@
+{ pkgs, stdenv }:
```

Comment:
`pkgs` shouldn't be passed as argument, instead pass each thing you need (e.g. `fetchFromGitHub gperf ...`